spec/features/event_types_spec.rb

```diff
@@ -32,7 +32,7 @@
       within('table#event_types') do
         expect(page.has_content?('Party')).to be true
         expect(page.has_content?('13042')).to be true
-        expect(page.has_content?('#e4e4e4')).to be true
+        expect(page.has_content?('#E4E4E4')).to be true
```

Comment:
Does that belong to this commit? I guess it belongs to 89662894b130c855d8e468e465f4f89add7bd89f, doesn't it?